template-parts/hero-stacked.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Template for Displaying Hero on Stacked Template
+ *
+ * @package info.*.edu
+ */
+
+$page_id = get_the_ID();
+
+$hero_title     = get_post_meta( $page_id, '_headline', true );
+$hero_sub_title = get_post_meta( $page_id, '_subheadline', true );
+$headings_tags  = [
+	'span' => [],
+];
+
+$hero_classes = ! is_page_template( 'page-templates/full-width-hero.php' ) ? ' col-lg-8 pr-md-0' : '';
+$copy_classes = ! is_page_template( 'page-templates/stacked.php' ) ? ' ml-md-4' : '';
```

Comment:
It looks like this file (`hero-stacked.php`) only gets called by `page-templates/stacked.php`?
Therefore these two check for `is_page_templates()` are not needed and can be used inline in the actual attributes.

If so, same goes for line 40-44 below in this file.